torch/linalg/__init__.py

```diff
@@ -1142,17 +1197,17 @@
              This non-uniqueness problem is even worse when the matrix has repeated singular values.
              In this case, one may multiply the associated singular vectors of `U` and `V` spanning
              the subspace by a rotation matrix and `the resulting vectors will span the same subspace`_.
-             As such, `U` and `Vh` are not continuous functions of the inputs. Furthermore, different
-             platforms, like NumPy, or inputs on different devices, may produce different
-             `U` and `Vh` tensors.
+             As this choice is not canonical, it happens that `U` and `V`  are not continuous
```

Comment:
see previous comments for a suggested simplification of this language

If we do want to keep language like this we might want to substitute "unique" for "canonical"